src/types/context.ts

```diff
@@ -3,7 +3,7 @@ import { Octokit } from "@octokit/rest";
 import { StartStopSettings } from "./plugin-input";
 import { createAdapters } from "../adapters";
 import { Env } from "./env";
-import { Logs } from "@ubiquity-dao/ubiquibot-logger/.";
+import { Logs } from "@ubiquity-dao/ubiquibot-logger/";
```

Comment:
```suggestion
import { Logs } from "@ubiquity-dao/ubiquibot-logger";
```

The slash should not be required right?

Reply:
Implies index.ts I presume

Reply:
> Implies index.ts I presume (which is not convention)

I don't understand what you mean here not convention

> The slash should not be required right?

No It's not. One single `index.ts` files collects the important exports and makes them available via `...logger(/ | /. | /index`)

Reply:
@0x4007 do you mean you'd rather have `/logger/logs` than just `/logger`?

Imo packages with many exports and distinct sections benefit more from have multiple entries (e.g. `permit-generation` (which needs fixed export-wise, haven't got to it yet and low priority)) 

But with only a few exports it's more convenient to have a single entry.